src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -179,9 +165,28 @@
                      :title "Open Settings"
                      :active   (= @route-name :settings)}
              [:> Settings]]
+
             [button {:on-click #(dispatch [:modal/toggle])
                      :title "Choose Database"}
-             [:> LibraryBooks]]
+             [:div {:style {:display "flex"}}
```

Comment:
This sequence of added divs and styles could possibly be refactored into a single style applied to the button itself, and using `::stylefy/manual [[:div {:display "flex"}...]] to style its children.